trade-finance-manager-api/src/graphql/helpers/amendment.helpers.js

```diff
@@ -64,80 +62,37 @@ const calculateUkefExposure = (facilityValueInGBP, coverPercentage) => {
   return null;
 };
 
-// checks for coverStartDate and returns based on deal-type as stored differently
-const dealTypeCoverStartDate = (facilitySnapshot) => {
-  const { coverStartDate } = facilitySnapshot;
+const findLatestCompletedAmendment = (amendments) => {
+  const amendmentsReversed = [...amendments].reverse();
```

Comment:
why do we need the amendments in reverse order?

Reply:
as we need the last item in the array with a tfm object for the most latest amendment.  findLast() doesn't work yet so this was best way to use find